### PR TITLE
Update changelog for 3.2.2 release

### PR DESCRIPTION
**Pull request by YuyaoHu**
Things to consider before you submit the PR:

* [x] Has `CHANGELOG.md` been updated?
* [ ] Are tests passing locally?
* [x] Are the files formatted correctly?
* [ ] Did you add unit tests if this modifies the Windows code?
* [ ] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

Update changelog for 3.3.0 release

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,13 +1,25 @@
...
 
 #### Android
 
 * **[Fix]** Fix reporting Xamarin.Android version.
 
+#### iOS
+
+* **[Feature]** Add support for XCFrameworks.
+* **[Improvement]** Use namespaced NSUserDefaults keys with the MSAppCenter prefix for all the keys set by the SDK. Fixed a few keys missing namespace.
+* **[Feature]** Swift Package Manager support for Analytics and Crashes modules.
```
Comment (MatkovIvan) [inline CHANGELOG.md:15]: Both features aren't related to .NET

Comment (YuyaoHu) [reply CHANGELOG.md]: @MatkovIvan I made the changes. please help to review. thanks.

Comment (YuyaoHu) [reply CHANGELOG.md]: So this release should be 3.2.2?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,13 +1,23 @@
...
 
 ### App Center
 
 #### Android
 
 * **[Fix]** Fix reporting Xamarin.Android version.
 
+#### iOS
+
+* **[Improvement]** Use namespaced NSUserDefaults keys with the MSAppCenter prefix for all the keys set by the SDK. Fixed a few keys missing namespace.
```
Comment (MatkovIvan) [inline CHANGELOG.md:13]:
```suggestion
* **[Improvement]** Use namespaced `NSUserDefaults` keys with the **MSAppCenter** prefix for all the keys set by the SDK. Fixed a few keys missing namespace.
```

**Pull request merged**